content/docs/user-guide/data-versioning.md

```diff
@@ -0,0 +1,154 @@
+# Data Versioning
+
+<!--
+## Data Versioning
+-->
+
+<admon type="tip">
+
+**Data version control** is the unifying trait across DVC features.
```

Comment:
> [Data Versioning](https://dvc-org-data-mgmt-cli-s-4sr5ax.herokuapp.com/doc/user-guide/data-versioning) page ([data-versioning.md](https://github.com/iterative/dvc.org/pull/4119/files#diff-d01612907e4ab14238625d537eaf42852d8566901d1bfe2fd3f2d4406a2d1dfc))
New

WDYT about introducing a guide page about _data versioning_ as the overarching concept and mechanism throughout DVC feature sets? That's probably the only relevant proposal left in this draft PR. Thanks

Cc @dberenbaum

Reply:
Sorry, I have been out of the loop on this PR, and I'm not sure I follow your comment or the current PR @jorgeorpinel. Right now I see a data management guide with several subpages and a separate data versioining page. Do we need two separate parts of the guide? Will people know when to go to each? What change are you suggesting in this comment?

Reply:
No worries @dberenbaum this was originally just an alternative to #4142 (working on that with @shcheklein).

> and a separate data versioining page. Do we need two separate parts of the guide

Indeed. That's the one I'm requesting comment on: Seeing that data versioning is the unifying element of all DVC features (IMO), could it help to have it's own page in the UG? The question can be further split into 2:

- Is data versioning (or version control for data) indeed what links or enables all other features? If so then it doesn't belong inside Data Mgmt, Pipelines, or Experiments (each one a section of the guide).
- Is it important enough to emphasize this or is it OK to just mention it in the existing guides as needed? (We do also have a use case about it).

> Will people know when to go to each?

The DV page should be clearly different from the [DM guide](https://dvc-org-guide-data-mgmt-yrwkeh.herokuapp.com/doc/user-guide/data-management). We can work on title and content to make sure.

> What change are you suggesting in this comment?

Nothing, just requesting comment on the DV page you can already see here. Thanks

Reply:
> * Is data versioning (or version control for data) indeed what links or enables all other features?

I think no. Take the recent experiments onboarding [blog post](https://iterative.ai/blog/exp-tracking-dvc-python), which doesn't require data versioning at all. Pipelines could do the same if users turn off cache for all outputs (and I think this is a reasonable direction to promote, especially for people who only want to check if their pipeline changed and are working with large amounts of data stored directly on cloud).

I think data versioning is a very important concept, but I think of it as the foundation for data management instead of as a separate concept.

Reply:
Makes sense. So then the Q is does it at least deserve its own page inside https://dvc.org/doc/user-guide/data-management ? Vs. sprinkling the concept in several pages (may be repetitive but easier to read). I guess we should plan the next step after https://github.com/iterative/dvc.org/pull/4042 (which is finally near finished) i.e. structure and content for the rest of the Data Management guide. Cc @shcheklein

Reply:
> So then the Q is does it at least deserve its own page inside https://dvc.org/doc/user-guide/data-management ?

IMO yes. I would start with this as the first page under data mgmt, or even put it in the index. It seems like all the other pages assume an understanding of data versioning.